frontend/src/containers/RecipePage/EditIcon/EditIcon.css

```diff
@@ -0,0 +1,107 @@
+#edit-icon-container {
+  display: inline-block;
+  float: right;
+  margin: 20px 26px;
+  border: 3px solid #ffa616;
+  border-radius: 13%;
+  color: #ffa616;
+  font-family: 'Nunito', sans-serif;
+  font-size: 14px;
+  height: 4em;
+  padding-left: .5em;
+  padding-right: 1.6em;
```

Comment:
Same with the em/px thing here. Maybe actually instead of creating separate components for these icons, we can just embed the icons directly, since we're going to use the icons8 stuff.